src/file-utils.ts

```diff
@@ -0,0 +1,43 @@
+import * as fs from 'fs';
+import type { CommandContext } from './commands/prompt/commands';
+
+const formatDate = (date: Date) => {
+  const day = date.getDate();
+  const month = date.getMonth() + 1; //Month from 0 to 11
+  const year = date.getFullYear();
+
+  return `${year}-${month < 10 ? '0' + month : month}-${day < 10 ? '0' + day : day}`;
+};
+
+const formatTime = (date: Date) => {
+  var hours = date.getHours();
+  var minutes = date.getMinutes();
+
+  return `${hours}-${minutes < 10 ? '0' + minutes : minutes}`;
+};
+
+const formatMessage = (message: string) => {
+  const base = message.split(' ', 5).join(' ');
+  return base.replace(/\/.*\\/i, '');
```

Comment:
Let's add a better escape function:

```
const escapeFilename = (str) => {
  return str.replace(/[\/\\:*?"<>|]/g, '_');
};
```

This is what `ai` suggested for escaping.

Reply:
Let's also add max 32 chars limit on the "first 5 words part"

Reply:
Done